frontend/src/components/avatar/avatar.jsx

```diff
@@ -48,8 +48,8 @@ function AvatarPhoto({id}) {
             
             axios.post(path, imagePost, {
                 headers: {
-                  'Authorization': 'token ' + team10token
-                  //'Authorization': 'Basic ' + team10Authorization
+                  'Authorization': 'token ' + JSON.parse(localStorage.getItem('user')).token
```

Comment:
Just be careful when doing `localStorage.getItem('user'))`.
It might cause a server error if the token does not actually exist.
HOWEVER, it should always exist if the user is on any page other than `/login` and `/register`